### ISSUE TITLE
Could not start the project

### ISSUE THREAD

**Issue by Gregoirevda** [TTomaszito/react-native-background-audio#1]
When I install this package on a create-react-native-app clean new project, it fails

**Comment by TuTomasz** [comment]
hey I haven't really updated this package in a while. I'm currently looking to rewrite it because it had some problems before. I'm curious what kind of error you get when you npm install it. Let me know

**Comment by Gregoirevda** [comment]
I had a bunch of errors: all where related to the fact that it couldn't resolve packages anymore in the 
`react-native-implementation.js` file. It's really linked to [this](https://github.com/facebook/react-native/issues/14209) issue. 

But Is it really possible to run JS in the background in expo? Because it is still an open feature (not finished) in Expo... 
https://expo.canny.io/feature-requests/p/audio-playback-in-background

**Comment by TuTomasz** [comment]
yeah they have been changing  alot in the expo platform making it hard to make these modules new errors pop up all the time. Well if you are looking to play audio in the background while the app is open then expo has some code for that in their docs.  What functionality are you trying to achieve?

**Comment by Gregoirevda** [comment]
Running music in the background, when your not in the app

**Comment by TuTomasz** [comment]
That cant be done using expo not yet at least. That has to be done in native objective c for now I think.

**Comment by bharathi-uitoux** [comment]
Hi there,
Any demo available to check how it works?
